rules/windows/persistence_runtime_run_key_startup_susp_procs.toml

```diff
@@ -1,14 +1,15 @@
 [metadata]
 creation_date = "2020/11/19"
 maturity = "production"
-updated_date = "2020/01/28"
+updated_date = "2020/02/16"
```

Comment:
```suggestion
updated_date = "2021/02/16"
```

Reply:
nice catch thanks. Looks like there were some bad dates from previous updates too, so I added a test for this, to at least ensure that the updated date is newer than creation